Sources/ViewModels/ALKConversationViewModel.swift

```diff
@@ -1203,10 +1203,16 @@ open class ALKConversationViewModel: NSObject, Localizable {
             }
             NSLog("messages loaded: ", messages)
             self.alMessages = messages.reversed() as! [ALMessage]
-            self.alMessageWrapper.addObject(toMessageArray: messages)
-            let models = self.alMessages.map { $0.messageModel }
-            self.messageModels = models
+            let contactService = ALContactService()
 
+            if let alContact = contactService.loadContact(byKey: "userId", value:  self.alMessages[0].to),
+              let role = alContact.roleType,
+              role !=  NSNumber.init(value: AL_BOT.rawValue) {
```

Comment:
We already have a function to get the role, use that. We can create one if it's not present.